src/rules/resolutions.ts

```diff
@@ -15,9 +15,24 @@ import * as core from '@actions/core';
 import { PackageJson } from 'type-fest';
 
 export const validateResolutions = (packageJson: PackageJson) => {
-  if (packageJson.resolutions) {
+  const ignoredResolutions = core.getMultilineInput('ignore-resolutions');
+
+  const skipIgnoreResolutions =
+    !Array.isArray(ignoredResolutions) || ignoredResolutions.length === 0;
+  if (packageJson.resolutions && skipIgnoreResolutions) {
     core.setFailed(
       'Resolutions may not be set. Please investigate the root cause of your dependency issues!'
     );
   }
+
+  if (packageJson.resolutions && Array.isArray(ignoredResolutions)) {
+    const resolutions = Object.keys(packageJson.resolutions);
+
+    const isMatching = resolutions.every(resolution => ignoredResolutions.includes(resolution));
+    if (!isMatching) {
```

Comment:
```suggestion
    const allResolutionsAreIgnored = resolutions.every(resolution => ignoredResolutions.includes(resolution));
    if (!allResolutionsAreIgnored) {
```

Reply:
Cheers, wasn't happy with the name either 🙃